src/atoms/Card/CardBody.js

```diff
@@ -7,6 +7,7 @@ import type { PropSizes } from '../../types';
 type CardBodyProps = {|
   children: React$Node,
   padding?: PropSizes,
+  isScrollable?: boolean,
```

Comment:
What are you think about renaming `isScrollable` to `scrollable`? `isXXX` usually used for methods/functions